assets/css/lighttheme.css

```diff
@@ -1,9 +1,16 @@
 a:hover,
 a:active {
-  color: #167ac6;
+  color: #167ac6 !important;
 }
 
 a {
   color: #61809b;
   text-decoration: none;
 }
+
+/* All links that do not fit with the default color goes here */
+a > .icon, 
+.pure-u-md-1-5 > .h-box > a[href^="/watch?"],
+.playlist-restricted > ol > li > a {
+  color: #303030;
+}
```

Comment:
I'm a little confused as to what is happening here. Would it make more sense to have only video links be the exception, and have `#303030` be the default color? Or I'm likely misunderstanding how the styling is applied.

Reply:
I have been using Invidious with the changes I made a week ago and it seems to me that the colors in the links of the playlists and related videos don't look good compared to those of the description, the comments or the videos that appear in the other pages (like the main page, the search, the channels, etc), that's why I only revert the change in those sections while the rest keeps the color `#61809b`.
If you have doubts you can try this adjustment in the style for a few days with an extension like [Stylus](https://github.com/openstyles/stylus) before making the final merge and if you are not convinced you can propose the changes you think appropriate.